content/docs/use-cases/experiment-tracking.md

```diff
@@ -0,0 +1,96 @@
+│ 7317bc6                 │ Jul 18, 2021 │       - │      - │ 10           │ 16               │
+│ ├── e2647ef [exp-ee8a4] │ 05:14 PM     │ 0.23146 │ 0.9145 │ 10           │ 64               │
+│ ├── 15c9451 [exp-a9be6] │ 05:14 PM     │ 0.25231 │ 0.9102 │ 10           │ 32               │
+│ ├── 9c32227 [exp-17dd9] │ 04:46 PM     │ 0.23687 │ 0.9167 │ 10           │ 256              │
+│ ├── 8a9cb15 [exp-29d93] │ 04:46 PM     │ 0.24459 │ 0.9134 │ 10           │ 128              │
+└─────────────────────────┴──────────────┴─────────┴────────┴──────────────┴──────────────────┘
+```
+
+![]() _Organized experiments in DVC_
+
+Unlike tools that focus on experiment navigation, DVC's approach guarantees
+reproducibility by integrating with Git directly (instead of saving fragile
+versioning metadata). This enables distributed collaboration along the way, via
+optional hosting like GitHub or GitLab.
+
+DVC projects are lightweight and local-first (no need for special servers or
+services). This means you control where and how your data is saved and shared.
+They can also make more efficient use of storage via <abbr>caching</abbr>, which
+prevents repetitive data transfers for every experiment run.
+
+On top of all that, DVC is completely language agnostic. You can expect the same
+convenience and performance whether you are using Jupyter Notebooks or Scala,
+CSV data frames or HDFS.
+
+|                   | DVC Experiments               | MLFlow                | Weights & Biases            | Neptune            |
+| ----------------- | ----------------------------- | --------------------- | --------------------------- | ------------------ |
+| UI                | Terminal<br/>+ [Web][studio]  | Web (local or hosted) | Web                         | Web                |
+| Infrastructure    | None (just Git)               | Web server            | SaaS                        | SaaS               |
+| Versioning        | Git (standard, robust)        | Metadata (fragile)    | Metadata (fragile)          | Metadata (fragile) |
+| Language Support  | Any<br/>(agnostic)            | Several (REST)        | Python only                 | Python only        |
+| Compute & Storage | User (yours)                  | User (yours)          | Theirs                      | Theirs             |
+| Collaboration     | Distributed<br/>(Git hosting) | Centralized           | Centralized                 | Centralized        |
+| Licensing         | Open Source                   | Open Source           | Proprietary<br/>+ OS client | Proprietary        |
+| Cost              | Free + [storage]              | Free [+ cloud][mp]    | [$$$][np]                   | [$$][wp]           |
+
+<!-- how are we defining 'Centralized' for collaboration? I think it might be stronger if we phrase it differently -->
+<!-- Pipeline integration is one key difference imo (maybe worth a new row?) . You would need to use complementary tool for pipelines in both MLFlow ans Wandb -->
```

Comment:
- do we not want to mention APIs? e.g. nascent https://dvc.org/doc/api-reference
- shouldn't we include more `[links](wherever possible)`?

Reply:
Oops those notes were not suppose to make it to the PR 😋 but yes, will link more.